src/Symfony/Component/HttpFoundation/Exception/SessionNotFoundException.php

```diff
@@ -20,7 +20,7 @@
  */
 class SessionNotFoundException extends \LogicException implements RequestExceptionInterface
 {
-    public function __construct($message = 'There is currently no session available.', $code = 0, \Throwable $previous = null)
+    public function __construct(string $message = 'There is currently no session available.', int|string $code = 0, \Throwable $previous = null)
```

Comment:
```suggestion
    public function __construct(string $message = 'There is currently no session available.', int $code = 0, \Throwable $previous = null)
```

`$code` should be an integer, see https://www.php.net/manual/de/exception.construct.php

Reply:
updated